### PR TITLE
Try to strengthen the wording on our MSRV

### PR DESCRIPTION
**Pull request by clux**
after discussion in https://github.com/kube-rs/kube-rs/discussions/921
Screenshot from 2022-05-26 20-04-30

didn't want to put down any greater number than 2 (or set 3 months) since it feels a bit wishy-washy when it's best effort

### CODE REVIEW

File: docs/rust-version.md [hunk 1]
```diff
@@ -4,11 +4,15 @@ Builds on CI always run against the most stable rust version, but we also suppor
...
+The MSRV is shown in the main [Cargo.toml.rust-version key](https://github.com/kube-rs/kube-rs/blob/80272c04ae0036983bb0de4d522003cfd1f5c189/kube/Cargo.toml#L15) plus as a [**readme badge**](https://github.com/kube-rs/kube-rs#kube-rs):
 
 > [![Rust 1.XX](https://img.shields.io/badge/MSRV-1.XX-dea584.svg)](https://github.com/rust-lang/rust/releases/)
 
-We have no official policy on when bumps occur, but we **try** to let the MSRV trail one or two stable versions behind the latest stable as a convenience to users downstream.
+Our MSRV policy is to **try** to let our MSRV **trail 2 stable versions behind the latest stable** as a convenience to users downstream.
+
+!!! note "Best Effort Policy"
+
+    Note that this policy, while sometimes more lenient than 2 version, it is also **not guaranteed to hold** due to dependencies we have to upgrade. The version shown should be taken as __best effort__ and __descriptive__, and it is verified as buildable by CI.
```
Comment (teozkr) [inline docs/rust-version.md:15]:
```suggestion
    Note that this policy, while sometimes more lenient than 2 versions, it is also **not guaranteed to hold** due to dependencies we have to upgrade. The version shown should be taken as __best effort__ and __descriptive__, and it is verified as buildable by CI.
```

File: docs/rust-version.md [hunk 2]
```diff
@@ -4,11 +4,15 @@ Builds on CI always run against the most stable rust version, but we also suppor
 
 ## Minimum Supported Rust Version
 
-The MSRV is the version listed in the badge at the [top of the main README](https://github.com/kube-rs/kube-rs#kube-rs), and it is verified as buildable as part of the main CI action.
+The MSRV is shown in the main [Cargo.toml.rust-version key](https://github.com/kube-rs/kube-rs/blob/80272c04ae0036983bb0de4d522003cfd1f5c189/kube/Cargo.toml#L15) plus as a [**readme badge**](https://github.com/kube-rs/kube-rs#kube-rs):
```
Comment (teozkr) [inline docs/rust-version.md:7]: Is the plan to keep this link up to date over time?

File: docs/rust-version.md [hunk 3]
```diff
@@ -40,7 +44,8 @@ CI always runs documentation and formatting builds against the latest `nightly`,
 rustup update nightly
 ```
 
-
+--8<-- "includes/abbreviations.md"
+--8<-- "includes/links.md"
```
Comment (teozkr) [inline docs/rust-version.md:48]: That's.. a syntax :P

**Review by teozkr — APPROVED** [review]

File: docs/rust-version.md (hunk 2 above)
Comment (clux) [reply docs/rust-version.md:7]: oh, hm, good point, maybe i should point it in the general direction without a hash.

File: docs/rust-version.md (hunk 3 above)
Comment (clux) [reply docs/rust-version.md:48]: it from the reasonably popular [pymdown-extensions](https://facelessuser.github.io/pymdown-extensions/extensions/snippets/#snippets-notation) which this mkdocs setup uses heavily.

but yeah. it's __something__ for sure.

File: docs/rust-version.md (hunk 2 above)
Comment (clux) [reply docs/rust-version.md]: done, simpler now.

**Pull request merged**